deploy/examples/operator-openshift.yaml

```diff
@@ -658,6 +658,9 @@ spec:
             - name: ROOK_ENABLE_MACHINE_DISRUPTION_BUDGET
               value: "false"
 
+            - name: ROOK_DISABLE_ADMISSION_CONTROLLER
```

Comment:
Per discussion, let's see if we can just change this default in the csv generation, but leave it enabled in the upstream example.

Reply:
done, but I have small doubt if the value field in env has `value: true` instead of `value: "true"` in CSV will not cause the problem right? confirm with Nitin he said we should be good

Reply:
The env var values are expected to be strings. If it doesn't have quotes, it would be interpreted as a bool, which could be slightly different.

Reply:
ocs-operator doesn't create rook operator deployment right? it is CSV that creates the rook operator

Reply:
> The env var values are expected to be strings. If it doesn't have quotes, it would be interpreted as a bool, which could be slightly different.

in YAML, it is different but when it is in CSV I think it will work at least I have noticed without quotes value in csv sometime.

Reply:
Yes correct, but this can be added here https://github.com/red-hat-storage/ocs-operator/blob/main/tools/csv-merger/csv-merger.go#L223 right?

Reply:
Is this used in csv generation for downstream builds?